web-registry/src/index.tsx

```diff
@@ -44,17 +48,19 @@ ReactDOM.render(
     cacheLocation="localstorage"
   >
     <AuthApolloProvider>
-      <IntercomProvider appId={intercomId} autoBoot>
-        <LocalizationProvider dateAdapter={AdapterDateFns}>
-          <WalletProvider>
-            <LedgerProvider>
-              <ThemeProvider injectFonts>
-                <App />
-              </ThemeProvider>
-            </LedgerProvider>
-          </WalletProvider>
-        </LocalizationProvider>
-      </IntercomProvider>
+      <QueryClientProvider client={queryClient}>
```

Comment:
New react-query provider, allow reusing all query results made with react-query across the app.